README.md

```diff
@@ -81,85 +95,61 @@ Heroku doesn't just fire up the server automatically, so you have to tell it to:
-
-- AWS_ACCESS_KEY_ID
-- AWS_SECRET_ACCESS_KEY
-- LAMBDA_ROLE
+### Setup
+Getting going on Lambda is pretty straightforward. First, you'll need to create an Amazon developer account if you don't have one already. After that, browse to the [IAM Management Console](https://console.aws.amazon.com/iam/home) where you will create a new user:
 
-The environment variables are stored in an `.env` file which you need to create yourself from a copy of `.env.lambda`
+![First page](http://i.imgur.com/rymHquZ.png)
+You can enter whatever username you want here, but make sure that Programmatic access is checked.
 
-**Take care with your `.env` file, it contains access details that you do not want to be uploaded to any public repositories (or pasted in a forum)**
+![Second page](http://i.imgur.com/TXX25BI.png)
+Attach the 'AdministratorAccess` permission to the new user.
 
-You do not need to change the options that already have values.
+![Third page](http://i.imgur.com/83o9cen.png)
+Confirmation page should look like this.
 
-If using **eu-west-1** AWS region to host your Lambda function then you can adjust `AWS_DEFAULT_REGION` accordingly.
+![Fourth page](http://i.imgur.com/3PZM92w.png)
+Once you get here, do not leave the page! You need the Access key ID and Secret access key for the next step. If you close now you'll have to create a new user. You can't get access to the Secret access key again.
 
-#### About AWS regions
+Next, run these commands to configure your computer for AWS service access:
+`pip install awscli` and then `aws configure`. Just follow the prompts, and copy paste the keys when it asks for them. When it asks for location, if you are in the US, enter: `us-east-1`, and if you are in Europe: `eu-west-1`.
 
-The AWS documentation ["Creating an AWS Lambda Function for a Custom Skill"](https://developer.amazon.com/public/solutions/alexa/alexa-skills-kit/docs/developing-an-alexa-skill-as-a-lambda-function) states:
+After you've done that, run `pip install virtualenv`. This is required for a later step.
 
->  _Lambda functions for Alexa skills must be hosted in the US East (N. Virginia) region. Currently, this is the only Lambda region the Alexa Skills Kit supports_.
+Now, clone my repo: `git clone https://github.com/m0ngr31/kodi-alexa.git` and `cd kodi-alexa`. Once you are inside the project directory, you're going to create a new "Virtual environement" and then activate it:
+`virtualenv venv` and `source venv/bin/activate` (if you are on Windows, that's `venv\Scripts\activate.bat` or `venv\Scripts\activate.ps1` for Powershell).
 
-The above statement is repeated many times throughout the docs.
+After successfull completion, run `pip install -r requirements.txt` and `pip install zappa`. Before you deploy, you need to copy the `.env.example` file to `.env` and enter the correct information for: KODI_ADDRESS, KODI_PORT, KODI_USERNAME, and KODI_PASSWORD. I'll go over the other variables in another section below.
```

Comment:
It might be a good idea to avoid calling out specific variables here and simply put the documentation in the .env.example file.